Added Open Web API Link

pcpinme opened the pull request:
# :clipboard: **Description**

Updated NPM dependencies to latest versions.
Added Web API link. #112 
 
# :white_check_mark: **Code Review Checklist**

- [x] I completed a complete self-review of my contribution.

<!--
  Please confirm you have tested the extension on all browsers and on at least one Dynamics version. We encourage testing the changes on as many different environments as possible.
-->

I tested the extension for the following browsers:
  - [x] Google Chrome
  - [x] Mozilla Firefox
  - [x] Microsoft Edge
  - [x] Microsoft Edge (Chromium)

I tested the extension on the following versions of the product:
 - [x] Dynamics 365 v9
 - [x] Dynamics 365 v8.2
 - [x] Dynamics CRM 2016
 - [x] Dynamics CRM 2015
 - [x] Dynamics CRM 2013

Commit:
Fix build

Commit:
Fix npm

Commit:
Web API URL

Commit:
Removing shrinkwrap

Commit:
Simple open of webapi url

Commit:
Moving to Form Actions

Commit:
Add API version check

Commit:
Remove API Notification

Commit:
User friendly error message.

mehmetseckin commented on src/js/ui/pane.js:
```diff
@@ -1205,6 +1205,27 @@ $(function () {
                 }
+            });
+			
+			$("#open-webapi").click(function () {
+                try {
+					
+					var apiVersion = Xrm.Utility.getGlobalContext().getVersion();
+					var shortVersion= apiVersion.substring(3, apiVersion.indexOf(".") -1);
+					
+					Xrm.Utility.getEntityMetadata(Xrm.Page.data.entity.getEntityName(),"")
+						.then(function(result){
+							var url = Xrm.Page.context.getClientUrl() + "/api/data/v" + shortVersion + "/" + result.EntitySetName +"(" + Xrm.Page.data.entity.getId() + ")";
+							url = url.replace("{", "").replace("}","");
+					
+							window.open(url,'_blank');
+							
+						});
+                    
+                } catch (e) {
+                    CrmPowerPane.UI.ShowNotification("An error occured opening the Web API URL for this record.");
+                }
```
Not sure if it's an editor/browser issue, but the indentation seems a bit off on the browser.

mehmetseckin commented on src/html/ui/pane.html:
```diff
@@ -46,6 +46,7 @@
                 <li class="crm-power-pane-subgroup" id="crm-diagnostics"><span><span class="icon crm-diagnostics-icon"></span>Crm Diagnostics</span></li>
                 <li class="crm-power-pane-subgroup" id="performance-center"><span><span class="icon performance-center-icon"></span>Performance Center</span></li>
                 <li class="crm-power-pane-subgroup" id="mobile-client"><span><span class="icon mobile-icon"></span>Mobile Client</span></li>
+				<li class="crm-power-pane-subgroup" id="open-webapi"><span><span class="icon record-url-icon"></span>Open Web API Record</span></li>
```
Again, the indentation seems a bit off on the browser, not sure if it's due to editor preferences.

pcpinme replied:
Had to fix using Notepad. Not sure what Notepad++ or VS Code is showing correctly. Thanks for the feedback.

mehmetseckin reviewed (changes requested):
Wow, thanks a lot for your contribution! This looks good, very nice work! The indentation seemed a little off in a couple of places, not sure if it's a browser/editor issue. Apart from that, awesome job! :shipit:

Commit:
Fix identation

Commit:
Fix indentation

Commit:
Moved to new column

onurmenal reviewed (approved):
Thank you so much for your contribution. It is great work!✨

Commit:
Fix indentation

pcpinme commented:
Fixed indentation properly and also extended the Navigation section across two columns.

Commit:
Fix Firefox warning

Reading manifest: Warning processing background.persistent: Event pages are not currently supported. This will run as a persistent background page.

pcpinme commented:
When does this get merged back?

onurmenal commented:
> When does this get merged back?

Hi Paul,  It will be tested and merged this week. If it passes tests, I will start the publishing process. 

pcpinme commented:
Hi @onurmenal  , what is the process to have this feature available in the production version in the Chrome web store?
